src/pkg/backup/details/details.go

```diff
@@ -516,6 +516,7 @@ type ExchangeInfo struct {
 	ItemType    ItemType  `json:"itemType,omitempty"`
 	Sender      string    `json:"sender,omitempty"`
 	Subject     string    `json:"subject,omitempty"`
+	Path        string    `json:"path,omitempty"`
```

Comment:
Nit: the term 'path' is overloaded, and should be namespaced in usage.
```suggestion
	ParentPath        string    `json:"parentPath,omitempty"`
```

Reply:
Resolved with [8762275](https://github.com/alcionai/corso/pull/2690/commits/876227596f30eb12936b1aecd037ca680acbf0c5)